templates/sssd.conf

```
@@ -0,0 +1,9 @@
+{{ ansible_managed | comment }}
```

```suggestion
{{ ansible_managed | comment }}
{{ "system_role:tlog" | comment(prefix="", postfix="") }}
```

System Roles now add role name to the generated config files to help customers identify that it was generated by System Roles and for future analysis